tools/klee/main.cpp

```diff
@@ -249,6 +266,58 @@ class KleeHandler : public InterpreterHandler {
   unsigned getNumPathsExplored() { return m_pathsExplored; }
   void incPathsExplored() { m_pathsExplored++; }
 
+  void incTotalDepthPathsExploredOnExit(unsigned currentDepth) {
+    m_totalDepthPathsExploredOnExit += currentDepth;
+  }
+
+  void incTotalInstructionsOnExit(unsigned totalInstruction) {
+    m_totalInstructionsOnExit += totalInstruction;
+  }
+  unsigned getTotalDepthPathsExploredOnExit() {
+    return m_totalDepthPathsExploredOnExit;
+  }
+  unsigned getTotalInstructionPathsExploredOnExit() {
+    return m_totalInstructionsOnExit;
+  }
+  void incTotalDepthPathsExploredOnEarly(unsigned currentDepth) {
+    m_totalDepthPathsExploredOnEarly += currentDepth;
+  }
+
+  void incTotalInstructionsOnEarly(unsigned totalInstruction) {
```

Comment:
I suggest renaming to `incTotalInstructionsDepthOnEarlyTermination`. Similarly to all similar methods.